_includes/tutorials/finding-distinct/ksql/code/tutorial-steps/dev/outputs/create-inputs/output-0.log

```diff
@@ -1,4 +1,4 @@
 Message        
-----------------
+
```

Comment:
Not sure these intermediate output files matter.  I guess we'll address this in a follow-up PR where remove the output files from source control.  This applies to several output files in this PR.